go.mod

```diff
@@ -9,9 +9,12 @@ replace (
 	github.com/docker/distribution => github.com/docker/distribution v2.8.2+incompatible // rancher-machine requires a replace is set
 	github.com/knative/pkg => github.com/rancher/pkg v0.0.0-20190514055449-b30ab9de040e
 	github.com/matryer/moq => github.com/rancher/moq v0.0.0-20200712062324-13d1f37d2d77
-
 	github.com/rancher/rancher/pkg/apis => ./pkg/apis
 	github.com/rancher/rancher/pkg/client => ./pkg/client
+	github.com/rancher/steve => /Users/RSAKALA/steve
+
+	github.com/rancher/wrangler/v3 => /Users/RSAKALA/wrangler
+
```

Comment:
Left over references to local directories.

Reply:
Removed them.